yo1k/sandbox/postgresql/pgexercises/readme.md

```diff
@@ -0,0 +1,11 @@
+Contains some solutions to programming exercises 
+from [pgexercises.com](https://pgexercises.com/gettingstarted.html)
```

Comment:
Shouldn't the URI be `https://www.pgexercises.com`? (note the `www` subdomain and the absence of the `/gettingstarted.html` part)

Reply:
Done.